lib/std/http/Server.zig

```diff
@@ -303,6 +303,51 @@ pub const Request = struct {
         return http.HeaderIterator.init(r.server.read_buffer[0..r.head_end]);
     }
 
+    /// Retrieves the value of a specified HTTP header from a Request object.
+    ///
+    /// This function searches for a header with a name matching the provided string,
+    /// ignoring case. If found, it returns the header's value. If not found, it
+    /// returns an empty string.
```

Comment:
That seems like odd design, why not return null and let the caller decide?

Reply:
I was basing it off how Go's header value retrieval function works. If its more Zig-like to return possible null then I will change.